src/damage.h

```diff
@@ -85,6 +86,19 @@ struct damage_instance {
     void deserialize( JsonIn & );
 };
 
+class damage_over_time_data
+{
+    public:
+        damage_type type;
+        time_duration duration;
+        std::vector<bodypart_str_id> bps;
+        int amount;
+
+        void load_DoT_data( const JsonObject &obj );
```

Comment:
```suggestion
        void load( const JsonObject &obj );
```
id you do this, then you can simplify the json loading below